core/sha1.js

```diff
@@ -36,6 +36,12 @@ sjcl.hash.sha1.prototype = {
    * @constant
    */
   blockSize: 512,
+
+  /**
+   * Maximal number of bits this function can hash.
+   * @constant
+   */
+  maxSafeLength: Math.pow(2, 53) - 1, //Number.MAX_SAFE_INTEGER
```

Comment:
why not just use the number here?
`9007199254740991`
Calculating this every time sjcl loads seems a lot of work.